examples/python/quickstart.py

```diff
@@ -0,0 +1,72 @@
+from pyspark import SparkContext
+from pyspark.sql import Column, DataFrame, SparkSession, SQLContext, functions
+from pyspark.sql.functions import *
+from py4j.java_collections import MapConverter
+from delta.tables import *
+import shutil
+import threading
+
+# Create SparkContext
+sc = SparkContext()
+sqlContext = SQLContext(sc)
+
+# Enable SQL for the current spark session.
+spark = SparkSession \
+    .builder \
+    .appName("...") \
```

Comment:
dont use "..." as its a bad pattern to put in example. use `"local[*]"` and `quickstart`

do the same for all examples.